src/mongo/client/dbclient.cpp

```diff
@@ -2110,16 +2127,18 @@ void DBClientBase::update(
         0, "update document exceeds maxBsonObjectSize", obj.objsize() <= getMaxBsonObjectSize());
     updates.enqueue(new UpdateWriteOperation(query.obj, obj, flags));
 
+    bool bypassDocumentValidation = flags & UpdateOption_BypassDocumentValidation;
+
     WriteResult writeResult;
-    _write(ns, updates.ops, true, wc, &writeResult);
+    _write(ns, updates.ops, true, bypassDocumentValidation, wc, &writeResult);
 }
 
 BulkOperationBuilder DBClientBase::initializeOrderedBulkOp(const std::string& ns) {
-    return BulkOperationBuilder(this, ns, true);
+    return BulkOperationBuilder(this, ns, true, false);
```

Comment:
You can use the constructor directly. i.e. `BulkOperationBuilder bulk(client, ns, true, true)`. See tests.